src/Mapper/UserMapper.php

```diff
@@ -97,12 +97,8 @@ protected function defaultKanvasProperties($user, object $userDto, array $contex
         $userDto->countries = $user->countries ?: null;
         $userDto->states = $user->states ?: null;
         $userDto->cities = $user->cities ?: null;
-        /**
-         * Properties we need to overwrite base on the
-         * current app and company the user is running.
-         */
-        $userDto->default_company = $user->getDefaultCompany()->getId();
-        $userDto->default_company_branch = $user->getDefaultCompany()->defaultBranch->getId();
+        $userDto->default_company = $user->default_company;
```

Comment:
@rwhite27 this should be calling thing directly from user_config allowing the user to be on different apps and multiple company and branches